tools/appimage.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+# This code is based on an example recipe from the AppImage project,
+# https://github.com/probonopd/AppImages/blob/master/recipes/powershell/Recipe
+# Copyright (c) 2016 Simon Peter
+# The license of this code and of https://github.com/probonopd/AppImages/raw/master/functions.sh
+# is the MIT License, see https://github.com/probonopd/AppImages/blob/master/LICENSE
+#
+# Generate AppImage, http://appimage.org
+#
+# The resulting PowerShell AppImage is known to run on
+# CentOS-7.0-1406-x86_64-GnomeLive.iso
+# CentOS-7-x86_64-LiveGNOME-1511.iso
+# Chromixium-1.5-amd64.iso
+# Fedora-Live-Workstation-x86_64-22-3.iso
+# Fedora-Live-Workstation-x86_64-23-10.iso
+# SL-72-x86_64-2016-02-03-LiveDVDgnome.iso
+# debian-live-8.0.0-amd64-xfce-desktop+nonfree.iso
+# debian-live-8.4.0-amd64-gnome-desktop.iso
+# elementary_OS_0.3_freya_amd64.iso
+# kali-linux-2.0-amd64.iso
+# kali-linux-2016.1-amd64.iso
+# kubuntu-14.04.4-desktop-amd64.iso
+# kubuntu-15.04-desktop-amd64.iso
+# kubuntu-16.04-desktop-amd64.iso
+# linuxmint-17.3-cinnamon-64bit.iso
+# neon-devedition-gitunstable-20160814-0806-amd64.iso
+# netrunner-17-64bit.iso
+# ubuntu-14.04.1-desktop-amd64.iso
+# ubuntu-16.04-desktop-amd64.iso
+# ubuntu-gnome-16.04-desktop-amd64.iso
+# ubuntu-mate-16.04-desktop-amd64.iso
+# xubuntu-16.04-desktop-amd64.iso
+
+APP=powershell
+
+mkdir -p ./$APP/$APP.AppDir/usr/lib
+
+cd ./$APP/
+
+wget -q https://github.com/probonopd/AppImages/raw/master/functions.sh -O ./functions.sh
```

Comment:
We shouldn't be fetching scripts from potentially untrusted sources as part of building our package - we should have copies of everything we need in our repo.